src/main/webapp/irc.html

```diff
@@ -211,11 +211,9 @@ <h1 class="mb-0 text-onelive font-weight-bold text-center">Frequently Asked Ques
 
 <section class="section pb-0 bg-gradient-white">
     <div class="container">
-        <p class="display-3">Researchers</p>
-
+        <p class="display-3 text-center" style="margin-bottom: 3rem">Researchers</p>
```

Comment:
You can use "mb-5" class instead of using inline css

Reply:
Fixed! Thanks @Piumal1999